fdbserver/worker.actor.cpp

```diff
@@ -1148,6 +1148,8 @@ ACTOR Future<Void> workerServer(
 				DUMPTOKEN( recruited.tlogRejoin );
 				DUMPTOKEN( recruited.changeCoordinators );
 				DUMPTOKEN( recruited.getCommitVersion );
+				DUMPTOKEN( recruited.getLiveCommittedVersion);
```

Comment:
obviously the previous change on backup forgot to `DUMPTOKEN(recruited.notifyBackupWorkerDone)`. Can you add it in this PR as well? 

FYI. @jzhou77

Reply:
Done.